common-app-lib/src/cmr/common_app/services/kms_fetcher.clj

```diff
@@ -74,6 +75,7 @@
   "Calls GCMD KMS endpoints to retrieve the keywords. Response is a map structured in the same way
   as used in the KMS cache."
   [context]
+  (def keyword-scheme :granule-data-format)
```

Comment:
Remove this line. Not sure why it is here.

Reply:
It was for debugging purposes and I just missed it. Fixed